### DISCUSSION TITLE
Various kinds of "notes"

### DISCUSSION THREAD

**Discussion by eosfor** [Lednerb/bilberry-hugo-theme#608] [Ideas]
Sometimes there is a need to make na emphasis. In different places i've seen different representations of this, like below 

![image](https://github.com/Lednerb/bilberry-hugo-theme/assets/9363027/92ceb8e7-6604-4e68-b269-bd09d87c5e86)

It would be create to have something like this, if it is possible. For example it could be like

```
> NOTE: Some note

> IMPORTANT: Some warning
```

And, i'm not a web developer, so i have no idea where to even start with this :D

**Comment by igor-baiborodine** [comment]
@eosfor, currently, Hugo does not support render hooks for these markdown elements. You can create your own shortcodes to display these elements within the content's markdown.
